servers/dsub/jobs/controllers/utils/labels.py

```diff
@@ -1,4 +1,4 @@
-def dsub_to_api(job):
+def get_labels(job):
```

Comment:
nit: I like the previous naming better I think, it's basically just stripping out things that are not labels in the API.

Reply:
Done.